packages/react/src/components/ButtonGroup/README.md

```diff
@@ -0,0 +1,77 @@
+# Button Group
+
+> The button group component is to be utilized within IBM.com for grouping two
+> Button components together.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ButtonGroup } from '@carbon/ibmdotcom-react';
+import '@carbon/ibmdotcom-styles/scss/components/buttongroup/_buttongroup.scss';
+function App() {
+  return;
+  <ButtonGroup
+    buttons={[
+      {
+        link: '',
+        copy: 'Primary action button',
+        renderIcon: 'ArrowDown',
+      },
+      {
+        link: '',
+        copy: 'Secondary action button',
+        renderIcon: 'ArrowRight',
+      },
+    ]}
+  />;
+}
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the button group styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+#### Feature Flags
+
+To utilize the following features, set the following variable's to `true` within
+your `.env` file or your application build settings.
+
+```
+BUTTON_GROUP=true
+```
+
+> See
+> [feature-flags.md](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/docs/feature-flags.md)
+> and
+> [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/.env.example)
+> for more information
+
+## Buttons
+
+| Name      | Description                                                                                    |
+| --------- | ---------------------------------------------------------------------------------------------- |
+| `buttons` | Array of button objects to render. Each object should contain `link`, `copy`, and `renderIcon` |
+
+## Stable selectors
+
+| Name                        | Description                                 |
+| --------------------------- | ------------------------------------------- |
+| `dds--buttongroup`          | `ButtonGroup` wrapper layer                 |
```

Comment:
This is nit picky thing, but does it make sense to add a dash to divide the two words and follow some of the other class naming we have going on `dotcom-shell`, `locale-button`. Just a thought. This alone shouldn't block it going in.

Reply:
Updated with the hyphen.